site/en/docs/webstore/group-publishers/index.md

```diff
@@ -1,115 +1,125 @@
-  re-link their own merchant accounts to the publisher on the dashboard page.
-- When you want to create new store items, you will have the opportunity to choose whether to make
-  them part of your personal publisher account or the group publisher account before you upload the
-  item.
+* You can only set up group publishing once and create one group publisher account.
+* You must be the owner or manager of a Google Group to link the group. The drop-down list on the dashboard page only shows groups that you're a direct member of (for example, if you are a member of Group A and Group A is a member of Group B, you are not a direct member of Group B, and Group B is not shown on the drop-down list).
+* Once you finish the group publishing setup, it can take up to 30 minutes for developers in your group to see the changes.
+* A developer can act on behalf of multiple publishers if the developer is a member of multiple Google Groups linked to publishers.
+* If a merchant account is needed or is already set up, any member of the group can choose to re-link their own merchant accounts to the publisher on the dashboard page.
+* Before you upload a new Chrome Web Store item, check that you have selected the publisher you want to own that item. This may be your personal publisher account or any group publisher account you belong to.
 
 ## Move existing items to a Group Publisher account
 
-Once you set up a Group Publisher account (or once you're added as a member of a Google Group linked
-to a Group Publisher account), you can transfer your own items to the Group Publisher.
+Once you set up a group publisher (or once you're added as a member of a Google Group linked to a group publisher), you can transfer your own items to the group publisher.
 
-Remember that once you move items to a Group Publisher account, you won't be able to move them back
-to your personal publishing account.
+<p class="note">Once you move items to a Group Publisher account, you can't move them back to your personal publishing account.</p>
 
-To transfer items to a Group Publisher account:
+To transfer items to a group publisher account you need to use the old developer dashboard:
 
-1.  Sign into your [Chrome Web Store dashboard page][3].
-2.  From the publisher selection drop-down at the top of the dashboard, select the Group Publisher
-    you want to transfer your items to. It will load the dashboard page for the selected Group
-    Publisher.
-3.  Select **Transfer existing item(s)**, next to the **Add new item** button. You'll see a page
-    with a list of items you own personally.
-4.  Select the item(s) you want to transfer, and click the **Transfer** link to the right of the
-    item.
-5.  Click **Transfer** in the confirmation box.
+1. Sign in to the [Chrome Web Store developer console](https://chrome.google.com/webstore/devconsole).
+1. To bring up the old developer dashboard, click **Show More** in the "Welcome" tile at the bottom-left, and then click **opt out**:
```

Comment:
please update, use the new developer console. And call it developer console rather than developer dashboard in all places. Dashboard only means the dashboard page, which is a single page. The entire developer portal is the console.

Reply:
Done